keyboards/wilba_tech/wt80_bc/config.h

```diff
@@ -0,0 +1,56 @@
+/* Copyright 2020 Jason Williams (Wilba)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x6582 // wilba.tech
+#define PRODUCT_ID      0x80B0 // 80-BC
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    wilba.tech
+#define PRODUCT         wilba.tech WT80-BC
```

Comment:
```suggestion
#define PRODUCT         WT80-BC
```

Reply:
This is how I define `PRODUCT` on my PCBs so they appear like this in connected devices.

Reply:
This will also produce an output like this everywhere else:
`6582:80b0 wilba.tech wilba.tech WT80-BC`

Reply:
Yes. The issue I have with it is, it's purely a Windows-ism. It also does some dumb things like using the manu/product strings of the *driver* instead of the device - in Device Manager, keyboards are listed as `(Standard keyboards) HID Keyboard Device`.

In lsusb, System Information.app, and particularly the Toolbox, it's going to look odd.

Reply:
Everywhere else being where exactly?

Reply:
My example was from `lsusb` specifically, but fauxpark mentioned a few others as well.

Reply:
> I strongly disagree with those instructions, because it is _only_ a Windows thing. Pretty much everywhere else, including the Toolbox, it will show up as eg. `wilba.tech wilba.tech WT60-D`.

It is not _only_ a Windows thing. OSX does _exactly the same thing_

![image](https://user-images.githubusercontent.com/10153817/81887952-a6656880-95e3-11ea-8358-6de2a5b2cc96.png)

So the two major operating systems (used by 95% of users) do it this way. "Everywhere else" being Linux's `lsusb` command and QMK Toolbox concatenate manufacturer and product, and the latter only if a feature is enabled.

Reply:
@Wilba6582 do you have any thoughts on QMK making the manufacturing string optional for situations like this? Would you still want to declare it here?

Reply:
@jackhumbert I did consider making `MANUFACTURER` empty, but then it would be missing on OSX where it's quite visible, and probably *somewhere* deep in Windows where I can't find it. The other idea I had was that QMK Toolbox could concatenate only if `PRODUCT` does not start with `MANUFACTURER`. Then it's only `lsusb` output that might "look weird" from a redundant concatenation.

I didn't expect to have a debate about this here, but this is not the first time this has been criticised, and I'm being asked to change this one case when all my other PCBs (and some other people's PCBs) are already in QMK this way. So I thought it was worth pointing out that this is not just bowing down to some Window-ism and what my reasoning is for this.

I concur that it would be cleaner and neater for the OS to concatenate `MANUFACTURER` and `PRODUCT` but when that doesn't _most of the time_, I'm just proposing to deal with that situation that exists as best we can.

Reply:
There was a point made in our internal discussion about this that Microsoft and Apple don't display the manufacturer string as prominently because they probably see it as less important than the actual description of the device. I'd have to agree with that but I also understand you wanting to put `wilba.tech` in the product string, given your naming scheme 😄

At the end of the day, I just want these strings to look good, even if they're concatenated. A good idea might also be to amend the VIA docs slightly to less highly recommend the duplication and point out that in some places these strings may be concatenated, so anyone adding VIA support to a board should be aware of that.

The last thing I want to say is that I've been looking at adding a custom string descriptor to retrieve the QMK version and build date, which the Toolbox would use to detect any QMK-powered devices connected, regardless of whether Console is enabled. As an aside VIA could perhaps also do a similar thing by having QMK provide some kind of unique ID to detect off of, instead of enforcing changes to the VID and PID.